tests/functional/amdModuleCircular.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+	<head lang="en">
+	<meta charset="UTF-8">
+	<title>AMD Loading Test</title>
+</head>
+
+<body>
+	<script src="../../loader.js"></script>
```

Comment:
Here is an example of using minimal indenting: https://github.com/SitePen/dgrid/blob/master/test/intern/functional/Tree.html